src/openloco/interop/interop.hpp

```diff
@@ -155,4 +156,5 @@ namespace openloco::interop
     void write_nop(uint32_t address, size_t count);
 
     void register_hooks();
+    void hook_stdcall(uint32_t address, void *fn);
```

Comment:
Should probably be renamed, as this doesn't really do more than just add a `JMP` to the new function.

Reply:
`register_jmp`?

Reply:
`write_jmp` is more consistent with the functions `write_ret` and `write_nop`.